tests/utils.py

```diff
@@ -12,6 +15,15 @@ def test_data_filepath(file_name: str) -> Path:
     return test_data_folder() / file_name
 
 
+def test_data_output_filepath(file_name: str, subfolder: str) -> Path:
+    """Return path to file tests output folder in temp directory"""
+    folder = Path(tempfile.gettempdir()) / "wrench-tests"
```

Comment:
I think generally it is preferred to always use a unique sub-folder in temp dir (e.g. using `tempfile.mkdtemp()`) but we can change that later if needed. It is certainly good to at least start with a clean directory, so we do not use output files from previous test runs by mistake)